examples/csharp/HelloPhi/Program.cs

```diff
@@ -80,23 +80,33 @@ void PrintUsage()
     generatorParams.SetInputSequences(sequences);
     if (option == 1) // Complete Output
     {
+        var watch = System.Diagnostics.Stopwatch.StartNew();
         var outputSequences = model.Generate(generatorParams);
         var outputString = tokenizer.Decode(outputSequences[0]);
-
+        watch.Stop();
+        var runTimeInSeconds = watch.Elapsed.TotalSeconds;
         Console.WriteLine("Output:");
         Console.WriteLine(outputString);
+        var totalTokens = outputSequences[0].Length;
+        Console.WriteLine($"Tokens: {totalTokens} Time: {runTimeInSeconds:0.00} Tokens per second: {totalTokens / runTimeInSeconds:0.00}");
     }
 
     else if (option == 2) //Streaming Output
     {
         using var tokenizerStream = tokenizer.CreateStream();
         using var generator = new Generator(model, generatorParams);
+        var watch = System.Diagnostics.Stopwatch.StartNew();
         while (!generator.IsDone())
         {
             generator.ComputeLogits();
             generator.GenerateNextToken();
             Console.Write(tokenizerStream.Decode(generator.GetSequence(0)[^1]));
         }
         Console.WriteLine();
+        watch.Stop();
+        var runTimeInSeconds = watch.Elapsed.TotalSeconds;
+        var outputSequence = generator.GetSequence(0);
+        var totalTokens = outputSequence.Length;
+        Console.WriteLine($"Tokens: {totalTokens} Time: {runTimeInSeconds:0.00} Tokens per second: {totalTokens / runTimeInSeconds:0.00}");
```

Comment:
Should the comment say what option it is, like 'Streaming Tokens:' so we know which one was timed?

Reply:
that's better. will make it "Streaming Tokens:" for clarity